drivers/eeprom/eeprom_simulator.c

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2019 Laczen
```

Comment:
Note that assigning copyrights would be done to either real people or companies. So if you want to write here a copyright notice for you, you would need to write your name, not your GH account name.

Reply:
Laczen is my company name, and also my github account name :smile:

Reply:
aa ,ok. If it is an actual company then nothing :)